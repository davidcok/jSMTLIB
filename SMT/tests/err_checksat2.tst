; check-sat before set-logic
(check-sat)
