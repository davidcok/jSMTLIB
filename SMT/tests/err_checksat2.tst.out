(check-sat)
^^^^^^^^^^^
(error "The logic must be set before a check-sat command is issued")
